commands/scan.md

```diff
@@ -148,6 +148,11 @@ redis 127.0.0.1:6379>
 
 As you can see most of the calls returned zero elements, but the last call where a `COUNT` of 1000 was used in order to force the command to do more scanning for that iteration.
 
+Note that when enabling `Redis cluster-mode`, using a pattern with hash tag that implies a specific slot can save complexity.
+Because any key matching the pattern will be in the same slot.
+Redis would only iterate over keys in the slot, rather than the whole database.
```

Comment:
Same as above.

```suggestion
When using [Redis Cluster](/docs/management/scaling/), the search is optimized for patterns that imply a single slot.
If a pattern can only match keys of one slot,
Redis only iterates over keys in that slot, rather than the whole database,
when searching for keys matching the pattern.
```